src/test/java/org/inferred/freebuilder/processor/GenericTypeSourceTest.java

```diff
@@ -178,6 +178,22 @@ public void testJ6() {
         "  }",
         "",
         "  /**",
+        "   * Returns true if the required property corresponding to",
+        "   * {@link Person#getName()} is set.",
+        "   */",
+        "  public boolean isPropertyNameSet() {",
```

Comment:
As per discussion on issue #41, I'm not happy to have these methods added: they can't be removed when a default is set; and they make customization harder. Use a try/catch loop to discard the IllegalStateException thrown by getName().